docset/winserver2022-ps/activedirectory/Add-ADGroupMember.md

```diff
@@ -146,6 +146,25 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisablePermissiveModify
+Group membership updates use permissive modify by default. This suppresses an error when adding a member that is already member of the group.
+When this parameter is used, an error "The specified account name is already a member of the group" is returned.
+
+This parameter is available in Windows Server 2019 with the September 2020 Updates.
```

Comment:
```suggestion
```

Reply:
Maybe this sentence should be removed.

Reply:
Hi @dariomws, Thanks for the feedback. I noticed the sentence was in Remove-ADGroupMember as well. I removed the sentence in both articles.

Thanks,
Pat